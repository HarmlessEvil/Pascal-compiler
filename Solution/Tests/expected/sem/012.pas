var
	x: Real;
	y: Integer;

begin
	x := x * y + 1;
end.