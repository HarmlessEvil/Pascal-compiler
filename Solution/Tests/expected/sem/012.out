the program
À--var
|  À--:
|  |  À--Real
|  |  À--:
|  |  |  À--x
|  À--:
|  |  À--Integer
|  |  À--:
|  |  |  À--y
À--block
|  À--:=
|  |  À--x
|  |  À--+
|  |  |  À--*
|  |  |  |  À--x
|  |  |  |  À--y
|  |  |  À--1
